compiler/src/iree/compiler/Codegen/Dialect/PCF/Transforms/Passes.td

```diff
@@ -9,4 +9,27 @@
 
 include "mlir/Pass/PassBase.td"
 
+def ConvertForallToLoopsPass : Pass<"iree-pcf-convert-forall-to-loops", ""> {
+  let summary = "Converts scf.forall ops to pcf.loop";
+  let description = [{
+    Test pass for converting `scf.forall` ops without mapping attributes to
+    `pcf.loop` ops with sequential scope.
+
+    The input is IR containing `scf.forall` ops with tensor results and
+    `tensor.parallel_insert_slice` terminators. Only forall ops without
+    mapping attributes are converted.
+
+    The output replaces each matching `scf.forall` with a `pcf.loop`:
+    - Iteration bounds come from the forall's upper/lower bounds and steps
+    - `tensor.parallel_insert_slice` ops become `pcf.write_slice` ops
+    - Shared output tensors become tied `pcf.sref` region arguments
+    - The scope is set to `#pcf.sequential` for sequential execution
```

Comment:
Inst conversion from `scf.forall` to a sequential scope making this serialized loop (unless I am missing the context of `#pcf.sequential`.

Reply:
It's noted at the beginning of this pass description, but this is just a test pass for converting `scf.forall` without mapping (i.e. unspecified order w/ unmapped workers) to `pcf.sequential` (means the same thing). Naming of `pcf.sequential` aside, we'll never actually use this pass. Actual use cases will live in different directories and just call the transformation + provide the appropriate scope for the mapping.